### ISSUE TITLE
Group capstone: set up your Kanban board

### ISSUE THREAD

**Issue by Rifat8080** [Rifat8080/Javascript-Group-Capstone#21]
<b>We followed these steps below :</b>

- Joined a Zoom call with my team helped each other.

- Created an empty repository for the group project.

- Followed the steps described in [curriculum-tools-copy-projects](https://github.com/microverseinc/curriculum-tools-copy-projects).

- Opened the GitHub project (Kanban board) in the team repo.

- Make sure that in the card titles : Student A, Student B, or Student C.

- Make sure that each card starts with [Xpt] - that number indicates the number of points that shows how complex the task is. 1 point = 1 hour of work.

- According to team members removed all cards with Student C in the title.

- Decided who in your team is the Student A, Student B, or Student C.

- Assigned cards to the correct members

<b>Here is the link of my GH project link of Kanban Board: </b>

- https://github.com/Rifat8080/Javascript-Group-Capstone/projects/1
- We are two members in our coding partners group

Thanks for your reviewing the regarding matter!🎯

**Comment by V-Blaze** [comment]
## Hi @Rifat8080  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

![giphy](https://user-images.githubusercontent.com/107632780/229314554-3700de13-2c00-46d5-bc77-46ad09af7467.gif)


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Well-detailed summary✔
-  Kanbarn board setup correctly👋 
-  ✔

 ## Required Changes ♻️
- [ ] You have done great setting up the kanban board. :+1: There is a step you need to complete before moving to the next milestone.
> Each of the team members should assign themselves to the correct cards. Decide who will be the assignee for the cards marked as group tasks. All cards on your Kanban board should have assignees.
- [ ] According to the project requirement, each card should be assigned to the correct student, Currently, your cards don't have assignees, you should see the profile image of the assignees if done correctly. I'll suggest you update this. :smile: 

- - Your Card:
![Screenshot from 2023-06-26 10-27-21](https://github.com/Rifat8080/Javascript-Group-Capstone/assets/107632780/7323ca0b-a95f-4e2a-b3e0-1878398dc476)

- - Correctly assigned:
![Screenshot from 2023-06-26 10-27-46](https://github.com/Rifat8080/Javascript-Group-Capstone/assets/107632780/db6f1dd6-4e94-4cb0-8d76-3ae6fbce6e8d)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by LionRouge1** [comment]
Hi **Team**, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

![done](https://media.giphy.com/media/3o7abKhOpu0NwenH3O/giphy.gif)

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
